osf/models/node.py

```diff
@@ -391,6 +394,21 @@ def parent_node(self):
                 return parent
         return None
 
+    @property
+    def storage_limit_status(self):
+        """ This should indicate if a node is at or over a certain storage threshold indicating a status."""
+
+        if self.storage_usage is None:
```

Comment:
This shouldn't throw an error. If a node has no files yet, `self.storage_usage is None` will return true, and this will throw the error.